pages/linux/postconf.md

```diff
@@ -0,0 +1,29 @@
+# postconf
+
+> Postfix coniguration utility.
+> This command displays the values of main.cf configuration parameters by default and warns about possible mis-typed parameter names. It can also change the main.cf configuration parameter values.
```

Comment:
```suggestion
> Postfix configuration utility.
> This command displays the values of the `main.cf` configuration parameters by default and warns about possible mistyped parameter names. It can also change the `main.cf` configuration parameter values.
```

Reply:
Done.